packages/design-system/components/inputs/Searchbar/__storybook__/Searchbar.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from '@storybook/addon-docs'
+
+<Meta title="@baseapp-frontend-template / inputs/Searchbar" />
```

Comment:
Title should be` @baseapp-frontend | designSystem/Inputs/Searchbar` and please change the stories title to match this, bec right now it's not under `inputs` folder. And please also remove the tags: ['autodocs'] and argTypes obj from the stories as well, we dont need that anymore